MANIFEST.in

```
@@ -0,0 +1,3 @@
+include honeybee_energy_standards/data/*.json
```

You can use recursive-include in this case:  https://docs.python.org/2/distutils/sourcedist.html#principle

Then this single like should include all the current and future json files

```suggestion
recursive-include honeybee_energy_standards/data *.json
```

---

I was considering doing that but I wasn't sure about it. Now that I know you support it, I will implement it with `recursive-include`